app/javascript/src/components/Dashboard/Notes/Note.jsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+
+import { Clock } from "neetoicons";
+import { Typography, Tag, Tooltip, Avatar } from "neetoui";
+
+import { IMG_URL } from "./constants";
+import NoteDropdown from "./DropdownMenu";
+
+const Note = ({ note }) => {
+  const createdString = "Created 2 hours ago";
```

Comment:
Try to move away from the mentality of naming variables as technical entities. Rather try to name the variables from a business perspective. Let's say you are added to a very very big codebase, somewhere you see a variable named `createdString`. Would you be able to easily figure out what exactly is business aspect of this variable and understand its relevance with once look? I think not.

If we had named this more sensibly from a business perspective, say something like `lastCreatedAt` and if you see this variable within the `Note.jsx`, then you don't have to take any mental effort per se to figure out what this variable is for.

Lemme know if have any queries regarding this point.

Reply:
Understood. I have changed the variable name.